src/olympia/devhub/tests/test_views.py

```diff
@@ -230,12 +230,12 @@ def test_no_sort_updated_filter_for_themes(self):
         addon = addon_factory(type=amo.ADDON_PERSONA)
         addon.addonuser_set.create(user=self.user_profile)
 
-        # There's no "updated" sort filter, so order by the default: "Name".
-        response = self.client.get(self.themes_url + '?sort=updated')
+        # There's no "updated" sort filter, so order by the default: "Created".
+        response = self.client.get(self.themes_url)
         doc = pq(response.content)
-        assert doc('#sorter li.selected').text() == 'Name'
+        assert doc('#sorter li.selected').text() == 'Created'
         sorts = doc('#sorter li a.opt')
-        assert not any('?sort=updated' in a.attrib['href'] for a in sorts)
+        assert not any('?sort=created' in a.attrib['href'] for a in sorts)
```

Comment:
Similarly, this is checking that we're not offering 'updated' as sorting option, so it doesn't need to be changed either.  (That's why you tests are failing on Travis, afaik)

Reply:
Thanks @eviljeff ! I overlooked at the test. Fixed